modular_zubbers/modules/modular_items/code/food_and_drinks/food/meatslab.dm

```diff
@@ -0,0 +1,7 @@
+/obj/item/food/meat/slab/human/mutant/feline
+	icon = 'modular_zubbers/icons/obj/food/meat.dmi'
+	icon_state = "adhomai_meat"
+	desc = "An adhomian delight"
+	tastes = list("furry" = 1, "meat" = 3)
+	foodtypes = MEAT | RAW | GORE
+	venue_value = FOOD_MEAT_MUTANT
```

Comment:
I have no idea when you added this but it is funny

Reply:
i got a few suggestions to add a few things to them and added a few small things like that